easybuild/easyconfigs/n/ncompress/ncompress-4.2.4.4-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,40 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'ncompress'
+version = '4.2.4.4'
+
+homepage = 'http://ncompress.sourceforge.net/'
+
+description = """
+ Compress is a fast, simple LZW file compressor. Compress does not have the
+ highest compression rate, but it is one of the fastest programs to compress
+ data. Compress is the defacto standard in the UNIX community for compressing
+ files.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['https://github.com/vapier/ncompress/archive/']
+sources = ['%(name)s-%(version)s.tar.gz']
```

Comment:
I wish there was a GITHUB_SOURCE template. :)  Github is always confusing.  I think there is a way (sometimes) to retrieve using the full SOURCE_TAR_GZ.  In my browser if I click the v4.2.4.4.tar.gz it somehow manages to save it using the SOURCE_TAR_GZ name (and not just the v%(version).tar.gz name.